src/style.css

```diff
@@ -1 +1,107 @@
-/* add styles here */
+html {
+  font-weight: 300;
+}
+
+body {
+  margin: 0;
+}
+
+.search__title--big {
+  margin-top: 20px;
+  width: 100%;
+  height: 70px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+}
+
+.search__title--big:hover {
+  margin-top: 20px;
+  width: 100%;
+  height: 70px;
+  border-radius: 4px;
+  box-shadow: 4px 0 4px 4px #00000040;
+  border: 1px solid #e1e7ed;
```

Comment:
The styles for `.search__title--big:hover` are mostly identical to `.search__title--big`. Consider removing redundant styles to improve maintainability. Only include styles that change on hover.